js/huobi.js

```diff
@@ -6435,7 +6435,7 @@ module.exports = class huobi extends Exchange {
         const omitted = this.omit (account, [ 'positions' ]);
         const positions = this.safeValue (account, 'positions');
         let position = undefined;
-        if (market['future'] && market['inverse']) {
+        if ((market['future'] && market['inverse']) || market['swap']) {
```

Comment:
@youknowone can you please elaborate a bit more? what does this fix?

Reply:
I can't reproduce the result because I don't have balance on huobi for now.
But as I remember, `BTC/USD:BTC-xxxxxx` is a future while `BTC/USDT:USDT` is a swap.
When called fetchPosition, it only counted future but not swap without the patch.

Reply:
@youknowone I just tested and seems to be ok using the `master` branch
```
CCXT v2.8.47
huobi.fetchPosition (ADA/USDT:USDT)
2023-02-27T10:35:29.652Z iteration 0 passed in 2828 ms

{
  info: {
    symbol: 'ADA',
    contract_code: 'ADA-USDT',
    volume: '1.000000000000000000',
    available: '0E-18',
    frozen: '1.000000000000000000',
    cost_open: '0.411712000000000000',
    cost_hold: '0.363219000000000000',
```

Reply:
Thank you for checking 👍